operator/hack/build-index-image.sh

```diff
@@ -91,7 +103,7 @@ function fetch_opm() {
   local -r arch=$(go env GOARCH) || true
 
   OPM="${BASE_DIR}/bin/opm-${OPM_VERSION}"
-  "${SCRIPT_DIR}/get-github-release.sh" --to "${OPM}" --from "https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/${os_name}-${arch}-opm"
+  "${SCRIPT_DIR}/get-github-release.sh" --to "${OPM}" --from "https://github.com/${OPM_ORG}/operator-registry/releases/download/v${OPM_VERSION}/${os_name}-${arch}-opm"
```

Comment:
> Does this take a fixed script from your personal github account?

It takes a binary built from the fork.

> WDYTH we should fork the repositories into the stack rocks organization instead of our personal accs?

In general I agree, though there is precedent in `go.mod` for personal orgs.
As mentioned in the TODO above, this is temporary.